doc/mgr/dashboard.rst

```diff
@@ -94,6 +97,45 @@ a list like so::
 Configuration
 -------------
 
+SSL/TLS Support
+^^^^^^^^^^^^^^^
+
+All HTTP connections to the dashboard are secured with SSL/TLS. 
+
+You can generate a self-signed certificate with the command::
+
+  $ ceph dashboard create-self-signed-cert
+
+Note that most web browsers will complain about self-signed certificates and
+require explicit confirmation before establishing a secure connection to the
+dashboard.
+
+To properly secure a deployment, a certificate that is signed by your
+organization's certificate authority should be used.
+
+For example, a key pair can be generated with a command similar to::
+
+  $ openssl req -new -nodes -x509 \
+    -subj "/O=IT/CN=ceph-mgr-dashboard" -days 3650 \
+    -keyout dashboard.key -out dashboard.crt -extensions v3_ca
+
+The ``dashboard.crt`` file should then be signed by your organization's CA
+(certificate authority). Once that is done, you can set it with::
+
+  $ ceph config-key set mgr mgr/dashboard/$name/crt -i dashboard.crt
```

Comment:
`s/config-key set mgr mgr/config-key set mgr/`

(The extra " mgr " token was for the "config set" command only)